Modules/System/WebDAV/src/Authorization/WebDAVBasicAuthorization.Codeunit.al

```diff
@@ -0,0 +1,13 @@
+codeunit 5680 "WebDAV Basic Authorization"
+{
+    Access = Public;
+
+    procedure GetWebDAVBasicAuth(Username: Text; Password: Text): Interface "WebDAV Authorization"
```

Comment:
Same thing is happening at the SharePoint Auth., the client secret is added in plain text.

This is a Basic Auth interface. If I add an isolated storage implementation I force everybody to use this isolated storage implemenation. 

Maybe I could force a Base64 encoding of the password? Would that be okay?

Reply:
Base64 is not a good security measure. Perhaps making the procedure in the public facade non-debuggable would be enough, since at that point, the process as far as this implementation goes should be reasonable secure.